packages/core/src/lib/waku_relay/index.ts

```diff
@@ -6,9 +6,9 @@ import {
 } from "@chainsafe/libp2p-gossipsub";
 import {
   PeerIdStr,
+  SignaturePolicy,
   TopicStr,
-} from "@chainsafe/libp2p-gossipsub/dist/src/types";
-import { SignaturePolicy } from "@chainsafe/libp2p-gossipsub/types";
+} from "@chainsafe/libp2p-gossipsub/types";
```

Comment:
not directly related to the scope of the PR but fixed this while I was debugging some circular dependency red herrings

Reply:
should it be `import type` ?

Reply:
Just the types of`PeerIdStr` and `TopicStr` can be imported, however, `SignaturePolicy` needs to be imported fully.